cupy/core/elementwise.pxi

```diff
@@ -549,7 +550,7 @@ cdef class ElementwiseKernel:
         kern = _get_elementwise_kernel(
             args_info, types, self.params, self.operation,
             self.name, self.preamble, self.kwargs)
-        kern.linear_launch(indexer.size, inout_args)
+        kern.linear_launch(indexer.size, inout_args, shared_mem=0, block_max_size=128, stream=stream)
```

Comment:
Cython does not support this form. https://mail.python.org/pipermail/cython-devel/2013-February/003342.html